S/StrAPI/Compat.toml

```diff
@@ -22,3 +22,7 @@ ModuleInterfaceTools = "0.1.7-0.2"
 ["0.2"]
 ModuleInterfaceTools = "0.2"
 julia = "1.0-1.3"
+
+["1.0"]
+ModuleInterfaceTools = "1.0"
+julia = "1.0-1.1"
```

Comment:
This is a bug https://github.com/JuliaComputing/Registrator.jl/issues/103 so please retrigger the bot when the linked issue is fixed.